_targets.R

```diff
@@ -21,14 +29,22 @@ list(
 
   # PULL SITE DATA HERE
   tar_map(
-    values = tibble(state_abb = states),
+    values = tibble(state_abb = states) %>%
+      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png",
+                                        state_abb)),
     tar_target(nwis_inventory,
-               filter(oldest_active_sites, state_cd == state_abb) ),
-    tar_target(nwis_data, get_site_data(site_info = nwis_inventory,
-                                        state = state_abb,
-                                        parameter = parameter))
-    # Insert step for tallying data here
-    # Insert step for plotting data here
+      filter(oldest_active_sites, state_cd == state_abb)),
+    tar_target(nwis_data,
+      get_site_data(site_info = nwis_inventory,
+                    state = state_abb,
+                    parameter = parameter)),
+    tar_target(site_tallies,
+      tally_site_obs(nwis_data)),
+    tar_target(plot_sites,
+      plot_site_data(out_file = state_plot_files,
+                     site_data = nwis_data,
+                     parameter = parameter)),
+    names = state_abb
```

Comment:
This doesn't affect how the pipeline runs, but from a readability standpoint, I like to have this `names` specification up above right after `values` is defined, so that someone encountering the pipeline can see how the target branches will be named before the targets are defined.

Reply:
I prefer that too! The issue requested that I add `names` as the final argument to `tar_map()`, so I did it that way. I've changed the argument order to be more readable per your suggestion. Thanks!

Reply:
Ah - good to know that's what the issue requests. Thanks for mentioning that!